include/flatbuffers/base.h

```diff
@@ -165,6 +165,10 @@ template<typename T> void WriteScalar(void *p, T t) {
   *reinterpret_cast<T *>(p) = EndianScalar(t);
 }
 
+static inline size_t RoundUp(size_t size, size_t round_size) {
```

Comment:
Actually it appears to be only used in one place now, so it can be inlined again :)